geomesa-tools/bin/geomesa-kafka

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env bash
+#
+# Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Apache License, Version 2.0 which
+# accompanies this distribution and is available at
+# http://www.opensource.org/licenses/apache2.0.php.
+#
+
+setGeoHome ()
```

Comment:
I played around with this in cloud local. What I believe I found was this:
1. You have to copy/paste the code to find your base dir, because you will need that in order to source the first env script
2. a geomesa-env would be a great idea. or a geomesa.conf type thing to set some basic versions.